File: C++ Programming/problems on string manipulation/problem103.cpp
/*

103.Write a program which accept string from user and toggle the case. 
	
		Input :   “Marvellous Multi OS” 
		Output :   mARVELLOUS mULTI os 
		
		Author : Milind Nalawade
		
		Date : 15/07/2021
		
		language : C++ programming

*/
using namespace std;
#include<iostream>
#include<string.h>

void Covert(char* arr)
{
	for (int i=0;i<strlen(arr);i++)
	{
		if((arr[i]>=97)&&(arr[i]<=122))
		{
			char temp=arr[i]-32;
			arr[i]=temp;
		}
		else if(arr[i]>=65&&arr[i]<=90)
		{
			char ch=arr[i]+32;
			arr[i]=ch;
		}
	}	
	cout<<" ";
	cout<<arr;
}

//Entry point function

int main()
{	
	char arr[20];
	cout<<"\n Enter your string : ";
	scanf("%[^'\n']s",arr);
	Covert(arr);
	return 0;
}		

File: Python Programming/problems on string manipulation/problem103.py
'''
103.Write a program which accept string from user and toggle the case. 
	
		Input :   “Marvellous Multi OS” 
		Output :   mARVELLOUS mULTI os 	
		
		Author : Milind Nalawade
		
		Date : 15/07/2021
		
		language : Python 3.9.0
'''

#====
#Logic
#====
def Convert(str):
	print(str.swapcase())
#=============
#Entry point function
#=============
def main():
	str=input("\n Enter your string : ")
	Convert(str);
#=====
#starter
#=====
if __name__=="__main__":
	main()
